README.md

```diff
@@ -622,8 +622,9 @@ IEEE Conf. Application-specific Systems, Architectures and Processors (ASAP), 20
 - <a name="Kepple20eccv"></a>Kepple, D.R., Lee, D., Prepsius, C., Isler, V., Park, I. M., Lee, D. D.,  
 *[Jointly learning visual motion and confidence from local patches in event cameras](https://www.ecva.net/papers/eccv_2020/papers_ECCV/papers/123510494.pdf)*,  
 European Conf. Computer Vision (ECCV), 2020. [Suppl. Mat.](https://www.ecva.net/papers/eccv_2020/papers_ECCV/papers/123510494-supp.pdf)
-- [Paredes-Valles et al., CVPR 2021](#ParedesValles21cvpr),  
-*Back to Event Basics: Self-Supervised Learning of Image Reconstruction for Event Cameras via Photometric Constancy.*
+- <a name="ParedesValles21cvpr"></a>Paredes-Valles, F., de Croon, G. C. H. E.,  
```

Comment:
There cannot be two equal labels ParedesValles21cvpr. One has to be the main entry and the other one calls it. See the example of Bardow CVPR 2016 on how to have a main entry and a secondary one. The secondary just points to the main one, without repeating the links, etc. so that it is easier to maintain

Reply:
Thanks for correction, i have fixed. Now only code link is added in this commit.